src/middleware/service.availability.middleware.ts

```diff
@@ -35,6 +37,14 @@ export const serviceAvailabilityMiddleware = (req: Request, res: Response, next:
     return res.render(SERVICE_OFFLINE_PAGE);
   }
 
+  if (
+    (req.path === UPDATE_CONTINUE_WITH_SAVED_FILING_URL || req.path.endsWith(`/${RESUME}`))
+    &&
+    !isActiveFeature(FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME)) {
```

Comment:
I don't understand the intention here. The UPDATE_CONTINUE_WITH_SAVED_FILING_URL  page should only be re-directed from the start of the journey. The landing URL is different when save&resume is enabled I believe and the only way to get to it is from "/update-an-overseas-entity/" path? If this is only an extra check to make sure that is the case then fine.

Reply:
Probably overly restrictive but as done by registration